src/js/_enqueues/wp/emoji.js

```diff
@@ -277,16 +277,7 @@
 			return twemoji.parse( object, params );
 		}
 
-		/**
-		 * Initialize our emoji support, and set up listeners.
-		 */
-		if ( settings ) {
```

Comment:
Since this script is loaded by the `wp-emoji-loader.js`, we know that `window._wpemojiSettings` will always be defined. Otherwise, if it is not, then the `load` function would never be executed. So this whole block of code doesn't make sense to me.